x-pack/plugins/ml/public/components/json_tooltip/tooltips.js

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export
+const getTooltips = () => ({
+  new_job_id: {
+    text: i18n.translate('xpack.ml.jsonTooltip.newJobIdTooltip', {
```

Comment:
I believe `jsonTooltip` is not good namespace in my opinion it is better to name it just `tooltip`